src/server/scripts/Northrend/AzjolNerub/AzjolNerub/boss_krikthir_the_gatewatcher.cpp

```diff
@@ -61,220 +118,303 @@ enum Yells
     SAY_SEND_GROUP                                = 5
 };
 
-Position const SpawnPoint[] =
-{
-    { 566.164f, 682.087f, 769.079f, 2.21657f  },
-    { 529.042f, 706.941f, 777.298f, 1.0821f   },
-    { 489.975f, 671.239f, 772.131f, 0.261799f },
-    { 488.556f, 692.95f,  771.764f, 4.88692f  },
-    { 553.34f,  640.387f, 777.419f, 1.20428f  },
-    { 517.486f, 706.398f, 777.335f, 5.35816f  },
-    { 504.01f,  637.693f, 777.479f, 0.506145f },
-    { 552.625f, 706.408f, 777.177f, 3.4383f   }
-};
-
-enum Events
-{
-    EVENT_SUMMON                                  = 1,
-    EVENT_MIND_FLAY,
-    EVENT_CURSE_FATIGUE
-};
-
 class boss_krik_thir : public CreatureScript
 {
     public:
         boss_krik_thir() : CreatureScript("boss_krik_thir") { }
 
         struct boss_krik_thirAI : public BossAI
         {
-            boss_krik_thirAI(Creature* creature) : BossAI(creature, DATA_KRIKTHIR_THE_GATEWATCHER) { }
+            public:
```

Comment:
public by default, remove this keyword to keep consistence with all other scripts